rest-api/config/config_stable.json

```diff
@@ -53,25 +53,18 @@
       },
       "description": "Massive subnet mask. see https://precisionmedicineinitiative.atlassian.net/browse/PD-2412"
     },
-    "awardee-ucirvine@all-of-us-ops-data-api-stable.iam.gserviceaccount.com": {
```

Comment:
I'm assuming this is intentionally being removed?

Reply:
yes. The ip was moved to awardee-california. Changes were made to service accounts without us being told.